tests/Type/LazyTypeLoaderTest.php

```diff
@@ -0,0 +1,395 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Type;
+
+use Exception;
+use GraphQL\Error\InvariantViolation;
+use GraphQL\Tests\PHPUnit\ArraySubsetAsserts;
+use GraphQL\Type\Definition\InputObjectType;
+use GraphQL\Type\Definition\InterfaceType;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Schema;
+use PHPUnit\Framework\TestCase;
+use stdClass;
+use Throwable;
+use function lcfirst;
+
+class LazyTypeLoaderTest extends TestCase
+{
+    use ArraySubsetAsserts;
+
+    /** @var ObjectType */
+    private $query;
+
+    /** @var ObjectType */
+    private $mutation;
+
+    /** @var callable */
+    private $node;
+
+    /** @var callable */
+    private $content;
+
+    /** @var callable */
+    private $blogStory;
+
+    /** @var callable */
+    private $postStoryMutation;
+
+    /** @var callable */
+    private $postStoryMutationInput;
+
+    /** @var callable */
+    private $typeLoader;
+
+    /** @var string[] */
+    private $calls;
+
+    /** @var Type[] */
+    private $loadedTypes = [];
+
+    protected function _lazyLoad(string $name)
```

Comment:
missing return typehint

Reply:
why not simply this?

```suggestion
    private function lazyLoad(string $name)
```

I see there's no sniff for that, I'll try to introduce it...

Reply:
51caeef 

> why not simply this?

No objection. c085169